buidl/test/test_musig.py

```diff
@@ -1,17 +1,153 @@
+                "d86e713462f55d13e8d0ff9a74c5523ca0ce8baaa339c75d59c91549b3aacd89",
+                "c0361dc4b1f78a4f52cc431dde2467205544918ddc350b911c094ca78af5c194",
+                "14a9f3a1730874d86afbc50a82297970714d332100a82e89b86f6185f16c24aa",
+                "4eb803c43cb72e5fba9f255261452eb62e3fc798159082fedf3bbadd9a081794",
+                "d0f19e0c0a13a1e05b25b679fe62b9f32794cc7729e39f8c9a880b5bb76e4072",
+            ],
+            [
+                "332f121ff348c45b917d429985cbacf6a02d29ddeb940006c78eb620c92e6e17",
+                "a6993d21fb59bdbd5d440d91ef63a6efddc23afc8fe378e8ab302de5a02ea6db",
+                "c1040a3e8557900d4ac0d39bd13d3e118d949ddc6d887af332c274d355abee0b",
+                "23befb7cef7a0f6780df8068538a1a0318c79c6ae9929f2ac151788b90381621",
+                "bcbc1fdfa3c994601a12c4903b7c13126770c91b764a2ae9665e1d5108b87cbe",
+                "2975571d6f7eeb695b31a9f362d938ca7cd6adc9f88ca16b8530d029c5c6b77e",
+            ],
+            [
+                "147760f9692a4ac0b6f3620a1a78439513747d80db0740257aeb629f53a0f020",
+                "a635e81e738bb6c82b86cf99bb1e66b67d66bef15127f1458e011853773d6134",
+                "3cbde74f668afc7770d3ea4ed105f578d8fa17676796c950001f9e81f61d8df0",
+                "deb885f923049d858f2f777fcbc0259449744214de7312ae952ea80fa8d0fed3",
+                "771f2f656e3e03c35b7fd5271eefaf07f2cf839db6aa5756dabb270417b6b75f",
+                "03e89d0c9c4374872b31e638a529088a810d0dbab4268d4438105e5ed6c0627a",
+            ],
+            [
+                "0d0ac5169c2949584546ee723bbd9d968957f2ae2c6c81831d49037e06f64cc9",
+                "cc402bb99c35310e635a5e62e1906dba5fb644a97c4d5774f7f683c2d786a772",
+                "12e0e04f53c38290c191841c1e86980f91babc53c806375512f23cd1a39af109",
+                "2b4a3f84469e280594c694a9d26e0fd108ef8a8dce8189a4efb0369f3e9c6dbd",
+                "ef33036853535ecf3b42ca7339028d8233dbe7f7eff1ece81e206db433f0f35c",
+                "942732e83fda0f976b2ed2bf63c9526c30089326a19792a1fb65a1a742d411c4",
+            ],
+            [
+                "13733d0dea513fa3e68579a478670400891b48ecec10eac48de621a632a9fa84",
+                "663bfccfa355c03e6d59da97618e19f86025b8feb3d18b9233ed66c4dae1407c",
+                "bed92b873b26573a665a53ada307331122d929b16592c679bcbe17c7d3e345d0",
+                "3ebb43e8f32a388c5fce9ca37b497b93565a8de3036b1a5a7530b78982abf9d2",
+                "ea7b73b9e415416e8784c2e2e1d9727446ba38e1bfafa3618b3c20cbb4102300",
+                "e148b34e6d2bb76c2da7efb63b91b69b6bd274aa734a7c837f07a285bce512bb",
+            ],
+        ]
+        for test in tests:
+            want = bytes.fromhex(test[-1])
+            points = [S256Point.parse_bip340(bytes.fromhex(h)) for h in test[:-1]]
+            tap_script = MuSigTapScript(points)
+            self.assertEqual(tap_script.point.bip340(), want)
+
     def test_single_leaf_multisig(self):
```

Comment:
In general better to add a new test-case (without dismantling any of the old one). I think you've eliminated the only test case that calls the `multi_leaf_tap_root` code path?

Reply:
`test_multi_leaf_multisig` uses `multi_leaf_tap_root`